Jellyfin Player/Library/Models/BaseItem.swift

```diff
@@ -97,7 +97,10 @@ protocol PlayableIteming: Codable {
 extension PlayableIteming {
 
     func imageUrl(with type: ImageType) -> URL? {
-        return URL(string: "http://server753.seedhost.eu:8096/emby/Items/\(id)/Images/\(type.rawValue)")
+        let server = ServerManager.currentServer
+        let baseUrl = server!.baseUrl
```

Comment:
A small suggestion, as `server!.baseUrl` can lead to some unwanted crashes in some scenarios if `server` is nil.

Maybe use a `guard let` as followed?
```swift
guard let baseUrl = ServerManager.currentServer?.baseUrl else { return nil }
```

Reply:
This is a good idea. But then I good suggest to make a new pull request applying this on the other similar instances.